projects/github.com/bcicen/ctop/package.yml

```diff
@@ -0,0 +1,31 @@
+distributable:
+  url: https://github.com/bcicen/ctop/archive/refs/tags/v{{version}}.tar.gz
+  strip-components: 1
+
+display-name: ctop.sh
```

Comment:
I didn't mean this. I meant the actual name of the package should be ctop.sh.

Reply:
so, bin/ctop.sh?

Reply:
if so, I'd disagree, since it's conventionally called using `ctop`, and we'd want to preserve that, no? your original comment was on the distributable line -- did you mean that we should be fetching the source from ctop.sh as well?

Reply:
no, `projects/ctop.sh/package.yml`